### PR TITLE
Refactoring to ad auction.md

### PR DESCRIPTION
**Pull request by lbdvt**
Moving to a "per process" documentation.

- Glossary moved to README, introducing key concepts
- publisher-requirement renamed to ad-auction
- README updated to reference ad-auction
- Workflows moved from workflow and dsp-implementation to ad-auction
- Transmission mechanisms (inc. parent/children) merged from dsp-implementation and publisher-requirement in ad-auction

### CODE REVIEW

File: mvp-spec/ad-auction.md [hunk 1]
```diff
@@ -1,25 +1,117 @@
...
+* **Transmission Requests**
+* **Transmission Responses**
+* **Transmission Results**
 
-Seeds, Transmission Requests, and Transmissions Responses are signed.
+### Seed
 
-Publishers are also required to:
-* Expose an Identity endpoint, so that participants can verify the the publisher's signature.
+A Seed links together the publisher site identity and a transaction id.
```
Comment (RomainLofaso) [inline mvp-spec/ad-auction.md:17]: "many transaction ids"

The new version of the specification states that an Addressable Content has a Transaction Id. A Seed is created for one or many Addressable Contents. Therefore, there are one or many Transaction Ids per Seed.

File: mvp-spec/ad-auction.md [hunk 2]
```diff
@@ -1,25 +1,117 @@
...
+### Transmission Requests
 
-### ad slots, Seeds, and Transmissions Requests
+A Transmission Request links together a Seed, a sender identity, and an array of Transmission Results (see below). 
+
+Participants sending User Id and Preferences with bid requests must include a Transmission Request alongside that communication.
+
+### Transmission Responses
+
+A Transmission Response links together a receiver identity, one or several transaction id, one or several content id, and a tree of Transmission Resulsts (see below).
```
Comment (RomainLofaso) [inline mvp-spec/ad-auction.md:27]: To be precise, there are zero to many *associations* between a Transaction Id and Content Id.

Comment (RomainLofaso) [inline mvp-spec/ad-auction.md:27]: "Transmission Results" without the typo at "Results"

Comment (RomainLofaso) [inline mvp-spec/ad-auction.md:27]: Do you think that we would be more descriptive of the functional aspects for all those definitions?

Example for the Transmission Response:
A Transmission Response is a response of a Receiver to a Sender for PAF Data. It expresses the commitment of the Receiver to respect the Preferences of the User for the given Identifiers thanks to a signature. Its contains:
* the identity of the Receiver
* the status of the Transmission (Equal 'Success' if the Receiver commits to use correctly the PAF Data).
* zero to many associations between a Transaction Id and Content Id depending if the Receiver offers potential Addressable Contents itself.
* A tree of Transmission Results that reflects the Transmissions instantiated thanks to the Receiver and that can participate in the Addressable Content.

File: mvp-spec/ad-auction.md (hunk 1 above)
Comment (lbdvt) [reply mvp-spec/ad-auction.md]: Ok, fixed

File: mvp-spec/ad-auction.md (hunk 2 above)
Comment (lbdvt) [reply mvp-spec/ad-auction.md]: Fixed

Comment (lbdvt) [reply mvp-spec/ad-auction.md]: Fixed.

Comment (lbdvt) [reply mvp-spec/ad-auction.md]: Yes, I think it's much better. 

FYI, I removed from the glossary:
- Receiver and Sender, as these terms are self-explanatory. I use them without capital letters if need be.
- PAF Data, replaced by User Id and Preferences, which is more self-explanatory.
- Addressable Content, replaced by "ad" or "content" for the same reasons

Transaction Id and Content Id are also not in the Glossary as of now, but they may have to be added.

Comment (lbdvt) [reply mvp-spec/ad-auction.md]: I updated the description of Seed, Transmission Request, Transmission Response

**Comment by RomainLofaso** [comment]
I think that we need to add a disclaimer at the beginning of at least the "Ad Auction" saying that this description relies on the version of the specifications where there is only one TransmissionId per Seed.

File: mvp-spec/ad-auction.md [hunk 3]
```diff
@@ -667,9 +762,9 @@ for nom.
...
 #### The OpenRTB Bid Response
 
-In step 5, the bidder (named Receiver in PAF Transmission) send back a 
+In **step 4**, the bidder sends back a 
 OpenRTB Bid Response. Each `bid` is associated with a Transaction Response. The 
-Transaction has the same structure explained in **Step 5** and is reachable in
+Transaction has the same structure as explained in **Step 4** and is reachable in
 the `ext` field of a `bid` (full path: `seatbid[].bid.ext.paf`).
 
 Here is an example:
```
Comment (RomainLofaso) [inline mvp-spec/ad-auction.md:770]: The examples are examples of the specifications for one Seed to one TransactionId.

**Review by RomainLofaso — APPROVED** [review]

**Pull request merged**